docs/apidocs/qiskit_nature.problems.second_quantization.lattice.models.rst

```diff
@@ -0,0 +1,17 @@
+﻿.. _qiskit_nature-problems-second_quantizatioin-lattice-models:
+
+.. automodule:: qiskit_nature.problems.second_quantization.lattice.models
+   :no-members:
+   :no-inherited-members:
+   :no-special-members:
+   
+   
+   
+
+   
+   
+   
+
+   
+   
+   
```

Comment:
Why is this trailing whitespace?

Reply:
Thank you for pointing this out. I deleted the whitespace.